quickstart/www/authoring/authoring-activities.php

```diff
@@ -0,0 +1,65 @@
+<?php
+    /**
+     * Copyright (c) 2017 Learnosity, MIT License
+     *
+     * Basic example of embedding Author API for creating Activities,
+     * using `mode: "activity_list"`.
+     */
+
+    // - - - - - - Section 1: server-side configuration - - - - - - //
+
+    // Include server side Learnosity SDK.
+    require_once __DIR__ . '/../../src/vendor/autoload.php';
+    use LearnositySdk\Request\Init as LearnosityInit;
+
+    // Author API configuration parameters.
+    $request = [
+        'mode' => 'activity_list', // Display the Activity list view on load.
+        'user' => [
+            'id'       => 'author123456' // Unique author id
+        ]
+    ];
+
+    // Public & private security keys required to access Learnosity APIs and
+    // data. These keys grant access to Learnosity's public demos account.
+    // Learnosity will provide keys for your own private account.
+    $consumerKey = 'yis0TYCu7U9V4o7M';
+    $consumerSecret = '74c5fd430cf1242a527f6223aebd42d30464be22';
+
+    // Parameters used to create security authorization.
+    $security = [
+        'domain'       => $_SERVER['SERVER_NAME'],
+        'consumer_key' => $consumerKey
+    ];
+
+    // Use Learnosity SDK to construct Author API configuration parameters,
+    // and sign them securely with the $security and $consumerSecret parameters.
+    $init = new LearnositySdk\Request\Init(
```

Comment:
```suggestion
    $init = new Init(
```

Reply:
Updated. I couldn't use your commit for some reason, so I replicated it manually.